extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.js

```diff
@@ -1247,10 +1253,11 @@ export class AmpLightboxGallery extends AMP.BaseElement {
 
   /**
    * Close gallery view
+   * @returns {!Promise}
    * @private
    */
   closeGallery_() {
-    this.vsync_.mutate(() => {
+    return this.vsync_.mutatePromise(() => {
```

Comment:
Should this be `this.mutateElement`? /cc @jridgewell

Reply:
I do need to refactor this entire component now...